products/public-websites/README.md

```diff
@@ -733,7 +735,6 @@ https://www.va.gov/discharge-upgrade-instructions/questions
 The discharge update wizard was written by the Decision Tools team. That team transitioned responsibility to the Search & Discovery Team, which has since become Public Websites purview. 
 
 **More info:**
-* vets.gov-team repo: [discharge tool issues](https://github.com/department-of-veterans-affairs/vets.gov-team/issues?q=label%3A%22discharge+tool%22+)
```

Comment:
Changed to `va.gov-team` and filtered issues by `Discharge Upgrade Wizard` label (`discharge tool` is not a label there anymore). No issues found.